src/phoenix/__init__.py

```diff
@@ -56,25 +51,5 @@
     "log_evaluations",
     "Client",
     "evals",
+    "run_experiment",
```

Comment:
maybe we don't hoist all the way so we leave room for making it a package eventually?

Reply:
sounds good